clients/clients.go

```diff
@@ -344,6 +344,12 @@ func (c *Clients) initClient(botClient *BotClient) error {
 	botClient.verificationSAS = &sync.Map{}
 
 	syncer := client.Syncer.(*mautrix.DefaultSyncer)
+	syncer.ParseErrorHandler = func(evt *mevt.Event, err error) bool {
+		// Events of type m.room.bot.options will be flagged as errors as this isn't an event type
```

Comment:
This is a bit miserable, but it seems to do the right thing. Presumably there isn't a way in which applications can tell the Syncer how an event type maps to a struct?

Reply:
afaict we're stuck with https://github.com/mautrix/go/blob/master/event/content.go#L19, referenced from https://github.com/mautrix/go/blob/master/event/content.go#L154

Reply:
You can just add stuff to `TypeMap`, that's why it's exported

Reply:
ah okay .. I wouldn't ordinarily dare to mutate a struct owned by a third party library .. !